notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb

```diff
@@ -8,7 +8,7 @@
       },
```

Comment:
<p>Title: Deploy Model -&gt; Deploy model</p><p> </p><p> </p><p>Deploy model to Google Cloud -&gt; </p><p>Next, deploy your Vertex AI Model resource to a Vertex AI Endpoint resource.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/825/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>